01-Subscription-Security/Readme.md

```diff
@@ -706,8 +706,27 @@ AzSK now supports the Privileged Identity Management (PIM) helper cmdlets. This
                                -ResourceName "AppServiceDemo" `
                                -DoNotOpenOutputFolder
       ```
+3. <h4> List Expiring assignments (-ListSoonToExpireAssignments) </h4>
 
-### Use Set-AzSKPIMConfiguration for configuring/changing PIM settings:
+      Use this command to list Azure role with PIM assignments at the specified scope that are about to expire in given number of days. Use respective parameters to list expiring assignments for a specific role, or to list assignments on a specific resource group or resource.
```

Comment:
list Azure roles
Use respective parameters to list expiring assignments for a specific role, resource group or resource.

Reply:
done